atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunnerImpl.java

```diff
@@ -124,10 +124,6 @@ public SweepResults run(TableReference tableRef, int batchSize, @Nullable byte[]
         // (2) force old writers to retry (note that we must roll back any uncommitted transactions that
         //     we encounter
         SweepStrategy sweepStrategy = sweepStrategyManager.get().getOrDefault(tableRef, SweepStrategy.CONSERVATIVE);
-        if (sweepStrategy == SweepStrategy.NOTHING) {
```

Comment:
Nope, just went through the code and it's a linear path. Shouldn't really slow anything down.